### DISCUSSION TITLE
算法太长，如何让设置如何不改变模板算法间距的情况下，设置算法跨页展示

### DISCUSSION THREAD

**Discussion by Ning-Ln** [note286/xduts#145] [Q&A] [answered]
![image](https://github.com/note286/xduts/assets/75731380/f6444f50-51b6-4556-8baf-27ae0e43cc52)

算法太长，如何让设置如何不改变模板算法间距的情况下，设置算法跨页展示，请版主帮下忙，马上要提交论文了，谢谢版主

**Comment by note286** [comment, marked as answer]
你的需求与xduts无关，这里我给出一个参考链接，如果不满足你的需求，请自行搜索。

[floats - Algorithm tag and page break - TeX - LaTeX Stack Exchange](https://tex.stackexchange.com/questions/33866/algorithm-tag-and-page-break/33869#33869)
